test/MapReduce.ts

```diff
@@ -61,12 +61,11 @@ describe("Model", () => {
         });
 
         it("should correctly map and reduce with model", () => {
+            let reducedModel = new Iridium.Model<Iridium.MapReducedDocument<string, number>, MapReducedInstance>(core, MapReducedInstance);
+            reducedModel.remove()
```

Comment:
You're gonna need to wait on this, so `return reducedModel.remove().then() => /* your map reduce test */`